### PR TITLE
v2.1

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: docs/interactions/Application_Commands.md [hunk 1]
```diff
@@ -998,6 +1039,9 @@ Fetches command permissions for a specific command for your application in a gui
 
 ## Edit Application Command Permissions % PUT /applications/{application.id#DOCS_RESOURCES_APPLICATION/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/{command.id#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object}/permissions
```
Comment (FedorLap2006) [inline docs/interactions/Application_Commands.md:1040]:
```suggestion
## Edit Guild Application Command Permissions % PUT /applications/{application.id#DOCS_RESOURCES_APPLICATION/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/{command.id#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object}/permissions
```

File: docs/interactions/Application_Commands.md [hunk 2]
```diff
@@ -1016,6 +1060,9 @@ Returns a [GuildApplicationCommandPermissions](#DOCS_INTERACTIONS_APPLICATION_CO
 
 ## Batch Edit Application Command Permissions % PUT /applications/{application.id#DOCS_RESOURCES_APPLICATION/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/permissions
```
Comment (FedorLap2006) [inline docs/interactions/Application_Commands.md:1061]:
```suggestion
## Batch Edit Guild Application Command Permissions % PUT /applications/{application.id#DOCS_RESOURCES_APPLICATION/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/permissions
```

**Review by FedorLap2006 — CHANGES_REQUESTED** [review]
The old endpoints have some naming issues, this PR could fix them as well

File: docs/interactions/Application_Commands.md (hunk 2 above)
Comment (Lulalaby) [reply docs/interactions/Application_Commands.md:1061]: Incorrect.

This includes global cmds too

Comment (Lulalaby) [reply docs/interactions/Application_Commands.md:1061]: or maybe yes.
but guild ac perm seems wrong due to them be referred to only guild cmds

File: docs/interactions/Application_Commands.md [hunk 3]
```diff
@@ -276,8 +276,8 @@ Application command permissions allow you to enable or disable commands for spec
 | USER    | 2     |
 | CHANNEL | 3     |
 
-> warn
-> This method is deprecated. See [Default Permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/default-permissions) for more info.
+> info
+> To set default permissions, see [Default Permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/default-permissions) for more info.
```
Comment (Lulalaby) [inline docs/interactions/Application_Commands.md:280]:
```suggestion
> To set default permissions, see [Default Permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/default-permissions).
```

**Pull request closed without merging**